src/Compilers/CSharp/Test/Symbol/Symbols/CovariantReturnTests.cs

```diff
@@ -27,6 +86,85 @@ private static void VerifyNoOverride(CSharpCompilation comp, string methodName)
             Assert.Null(overridden);
         }
 
+        /// <summary>
+        /// Verify that all assignments in the compilation's source have the same type and converted type.
+        /// </summary>
+        private static void VerifyAssignments(CSharpCompilation comp)
```

Comment:
>VerifyAssignments [](start = 28, length = 17)

Consider adding an assert that we found something to verify

Reply:
That is done in the following PR.

---
In reply to: [422132883](https://github.com/dotnet/roslyn/pull/43797#discussion_r422132883) [](ancestors = 422132883)